cmd/backup/script.go

```diff
@@ -622,3 +628,14 @@ func (s *script) must(err error) {
 		panic(err)
 	}
 }
+
+// skipPrune returns true if the given backend name is contained in the
+// list of skipped backends.
+func skipPrune(name string, skippedBackends []string) bool {
```

Comment:
Not sure if the name should be as specific, as the logic in the function really is very generic. Maybe this could be `containedEqualFold(item string, list []string)`?

Reply:
Thought about this too, also had this briefly inside `util.go` as it is very generic and could be used by any module.